pkg/framework/spec.go

```diff
@@ -20,10 +20,11 @@ type Test struct {
 	Values RawDict `json:"values,omitempty" yaml:"values,omitempty"`
 	Set    RawDict `json:"set,omitempty" yaml:"set,omitempty"`
 
-	Defs         string            `json:"defs,omitempty" yaml:"defs,omitempty"`
-	Release      *ReleaseSpec      `json:"release,omitempty" yaml:"release,omitempty"`
-	Server       *ServerSpec       `json:"server,omitempty" yaml:"server,omitempty"`
-	Capabilities *CapabilitiesSpec `json:"capabilities,omitempty" yaml:"capabilities,omitempty"`
+	Defs         string                   `json:"defs,omitempty" yaml:"defs,omitempty"`
+	Release      *ReleaseSpec             `json:"release,omitempty" yaml:"release,omitempty"`
+	Server       *ServerSpec              `json:"server,omitempty" yaml:"server,omitempty"`
+	Capabilities *CapabilitiesSpec        `json:"capabilities,omitempty" yaml:"capabilities,omitempty"`
+	Objects      []map[string]interface{} `json:"objects,omitempty" yaml:"objects,omitempty"`
```

Comment:
Would it be better if `Objects` was part of `ServerSpec`? :thinking:

Reply:
+1 Yes, I would prefer it under `Server` too. That makes it more intuitive for me.